templates/vhost/_file_footer.epp

```diff
@@ -0,0 +1,9 @@
+<% $define.each | $k, $v| {-%>
+  Undefine <%= $k %>
+<% } -%>
+</VirtualHost>
+<% if $passenger_pre_start {-%>
+  <%- [$passenger_pre_start].flatten().filter |$value| { $value }.each |$passenger_pre_start| {-%>
```

Comment:
The original variable name is `$passenger_pre_start` and we are also creating another local variable in scope with same name, which increases the readability. Can we name it something diff. Eg : 
```
<% if $passenger_pre_start { -%>
  <%- [$passenger_pre_start].flatten().filter |$value| { $value }.each |$value| { -%>
PassengerPreStart <%= $value %>
  <%- } -%>
<% } -%>
```

Reply:
done